examples/motoko_examples/http_counter/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "http_counter",
+  "version": "0.1.0",
+  "description": "Reimplementation of the [Motoko Http Counter example](https://github.com/dfinity/examples/tree/master/motoko/http_counter)",
+  "main": "index.js",
+  "scripts": {
+    "test": "ts-node --transpile-only --ignore=false test/test.ts"
+  },
+  "dependencies": {
+    "azle": "^0.4.0",
```

Comment:
Please remove all carots `^` and merge with main and run prettier, `npx prettier --write .`